src/webview/recipe.ts

```diff
@@ -124,8 +124,14 @@ contextBridge.exposeInMainWorld('ferdium', {
     safeParseInt(text),
   setDialogTitle: (title: string | null | undefined) =>
     dialogTitleHandler.setDialogTitle(title),
-  displayNotification: (title: string, options: any) =>
-    notificationsHandler.displayNotification(title, options),
+  displayNotification: (title: string, options: any) => {
+    notificationsHandler.displayNotification(
+      title,
+      // The following line is needed so that a proper clone of the "options" object is made.
+      // This line was causing issues with some services.
+      JSON.parse(JSON.stringify(options)),
```

Comment:
imo, it might be a good idea to move this fix into the jsUtils file and add a test for the same. wdyt? (especially if its repeated in multiple places in the codebase)

Reply:
Totally agree!

Just noticed we already have a `cleanseJSObject` util for that. Just changed the code.

Thank you @vraravam !